lambdas/process-journey-event/src/main/java/uk/gov/di/ipv/core/processjourneyevent/statemachine/stepresponses/PageStepResponse.java

```diff
@@ -12,11 +14,14 @@
 public class PageStepResponse implements StepResponse {
 
     private String pageId;
-    private String context = "";
+    private String context;
+    @Getter private Boolean mitigationStart;
 
     public Map<String, Object> value() {
-        return Map.of(
-                "page", pageId,
-                "context", context);
+        Map<String, Object> response = new HashMap<>();
+        response.put("page", pageId);
+        response.put("context", context);
```

Comment:
Why has this changed - do you need the response to be mutable?

Reply:
I changes it so the context wouldn't have to be initialised. Map.of requires non-null values